content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: XXXX'
+date: 2021-04-08
+slug: kubernetes-1-21-release-announcement
+---
+
+**Authors:** [Kubernetes 1.21 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release_team.md)
+
+We’re pleased to announce the release of Kubernetes 1.21, our first release of 2021!
+This release consists of 49 enhancements: 15 enhancements have graduated to stable, 15 enhancements are moving to beta, and 19 enhancements are entering alpha.
+
+## Major Themes
+
+### CronJobs Graduate to Stable!
+[CronJobs](/docs/concepts/workloads/controllers/cron-jobs/) (previously ScheduledJobs) has been a beta feature since Kubernetes 1.8! With 1.21 we get to finally see this widely used API graduate to stable.
+
+CronJobs are meant for performing all time-related actions, namely backups, report generation and so on. Each of these tasks should be allowed to run repeatedly (once a day/month, etc.) or once at a given point in time.
+
+### Immutable Secrets and ConfigMaps
+[Immutable Secrets](/docs/concepts/configuration/secret/#secret-immutable) and [ConfigMaps](/docs/concepts/configuration/configmap/#configmap-immutable) adds a new field to those resource types which will reject changes to those objects if set. Secrets and ConfigMaps by default are mutable which is beneficial for pods that are able to consume changes. Mutating Secrets and ConfigMaps can also cause problems if a bad configuration is pushed for pods that use them.
+
+By marking Secrets and ConfigMaps as immutable you can be sure your application configuration won't change. If you want to make changes you'll need to create a new, uniquly named Secret or ConfigMap and deploy a new pod to consume that resource. Immutable resources also have scaling benefits because controllers do not need to poll the API server to watch for changes.
+This feature is now graduating to stable in Kubernetes 1.21.
+
+### IPv4/IPv6 dual-stack support
+IP addresses are a consumable resource that cluster operators and administrators need to make sure are not exhausted. Having dual-stack support enables native IPv6 routing to pods and services, and it also improves a possible scaling limitation for workloads.
```

Comment:
```suggestion
IP addresses are a consumable resource that cluster operators and administrators need to make sure are not exhausted. In particular, public IPv4 addresses are now scarce. Having dual-stack support enables native IPv6 routing to pods and services, whilst still allowing your cluster to talk IPv4 where needed. Dual-stack cluster networking also improves a possible scaling limitation for workloads.
```
(rewording because single-stack IPv6 was already possible, but without being able to talk to the IPv4 internet it wasn't particularly useful).

Reply:
Has been addressed in the latest commit, thank you so much! :)